README.md

```diff
@@ -102,6 +103,7 @@ See the [_js](_js) directory for more information.
 [bash]: https://www.gnu.org/software/bash/
 [crux]: https://github.com/6c37/crux-ports-git/tree/3.3/shfmt
 [dockerised-shfmt]: https://hub.docker.com/r/jamesmstone/shfmt/
+[dockerized-shfmt]: https://github.com/PeterDaveHello/dockerized-shfmt
```

Comment:
The change proposed above will also fix these links. You can't have two named links with the same name.

Reply:
I tried to use dockeri**z**ed-shfmt than dockeri**s**ed-shfmt here :smile: Now revised.